src/h.js

```diff
@@ -1,3 +1,25 @@
+var nodePool = []
+var nodePoolHead = -1
```

Comment:
@andyrj @ngryman Is it alright to share the nodepool with all apps that use `h`? 🤔